queue/src/fiskaltrust.Middleware.Contracts/Repositories/FR/IJournalFRCopyPayloadRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using fiskaltrust.Middleware.Contracts.Repositories.FR.TempSpace;
+
+namespace fiskaltrust.Middleware.Contracts.Repositories.FR
+{
+    public interface IJournalFRCopyPayloadRepository
+    {
+        int JournalFRGetCountOfCopies(string cbPreviousReceiptReference);
+
+        bool InsertJournalFRCopyPayload(ftJournalFRCopyPayload c);
+
+        bool HasJournalFRCopyPayloads();
```

Comment:
```suggestion
        Task<int> GetCountOfCopiesAsync(string cbPreviousReceiptReference);
        Task<bool> InsertAsync(ftJournalFRCopyPayload c);
        Task<bool> HasEntriesAsync();
```

In 1.3 we should makt these async functions. Also we can simplify the method names because `JournalFRCopyPayload` is already in the interface name here 😁